yarn.lock

```diff
@@ -3285,6 +3285,13 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@lit-labs/react@npm:^1.1.1":
```

Comment:
Can you re-run `yarn` and check if there are diffs in `yarn.lock`? I was testing on my side and it picked a few `^` for `@lit-labs/react`